packages/components/src/toggle-group-control/toggle-group-control/styles.ts

```diff
@@ -57,21 +52,20 @@ const enclosingBorders = ( isBlock: ToggleGroupControlProps[ 'isBlock' ] ) => {
 };
 
 export const toggleGroupControlSize = (
-	size: NonNullable< ToggleGroupControlProps[ 'size' ] >,
-	__next40pxDefaultSize: ToggleGroupControlProps[ '__next40pxDefaultSize' ]
+	size: NonNullable< ToggleGroupControlProps[ 'size' ] >
 ) => {
-	const heights = {
-		default: '40px',
-		'__unstable-large': '40px',
+	const styles = {
+		default: css`
+			min-height: 36px;
+			padding: 2px;
+		`,
+		'__unstable-large': css`
+			min-height: 40px;
+			padding: 3px;
+		`,
```

Comment:
I am tempted to simplify this down to a

```js
if ( size === 'default` ) {
  return css`/* ... */`;
}
return css`/* ... */`;
```
but I'm going to keep this size variant object around because I think we'll be asked to make a 32px compact size variant soon.